invenio_jobs/services/config.py

```diff
@@ -73,3 +74,36 @@ class JobsServiceConfig(ServiceConfig, ConfiguratorMixin):
     }
 
     links_search = pagination_links("{+api}/jobs{?args*}")
+
+
+class RunsServiceConfig(ServiceConfig, ConfiguratorMixin):
+    """Service factory configuration."""
+
+    # Common configuration
+    service_id = "runs"
+    permission_policy_cls = RunPermissionPolicy
+
+    # TODO: See if we need to define custom Job result item and list classes
+    result_item_cls = RecordItem
+    result_list_cls = RecordList
+
+    # Record specific configuration
+    record_cls = Run
+
+    # TODO: See if these are needed since we don't index jobs
+    # indexer_cls = None
+    # indexer_queue_name = None
+    # index_dumper = None
+
+    # Search configuration
+    search = JobSearchOptions
+
+    # Service schema
+    schema = JobSchema
+
+    links_item = {
+        "self": JobLink("{+api}/jobs/{id}/runs"),
+        "start": JobLink("{+api}/jobs/{id}/runs"),
+        "delete": JobLink("{+api}/jobs/{id}/runs/{run_id}"),
+        "logs": JobLink("{+api}/jobs/{id}/runs/{run_id}/logs"),
```

Comment:
```suggestion
        "self": JobLink("{+api}/jobs/{job_id}/runs/{run_id}"),
        "stop": JobLink("{+api}/jobs/{job_id}/runs/{run_id}/actions/stop"),
        "logs": JobLink("{+api}/jobs/{job_id}/runs/{run_id}/logs"),
```

- Renamed `id` to `job_id`
- Added `stop` link

Reply:
Fixed.